extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -154,6 +154,15 @@ private function renderInfoboxHero( $data ) {
 	private function renderItem( $type, array $data ) {
 		$helper = new PortableInfoboxRenderServiceHelper();
 
+		if($type === 'image' && count($data) === 1) {
+			$data = $data[0];
+		} else if($type === 'image' && count($data) > 1) {
+			return $this->renderItem(
```

Comment:
More than one return statement in a function is a redflag - rethink this approach please.